### PR TITLE
Add more debug tips

### PR DESCRIPTION
**Pull request by fxiang1**
Changes
- Add debugging tips

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+When using breakpoints in VSCode you can only debug up to https://github.com/stolostron/cluster-curator-controller/blob/main/pkg/controller/launcher/job.go. For the code after that point, you still need to do scaffolding ie. use log or printf statements to debug because that code is called by the Job CRs which we can’t step through using VSCode.
+
+## Patching the hub cluster
+To patch the hub cluster you need to find the multicluster-engine `clusterserviceversion` or CSV. The easiest way I find is to look at the installed operators in the OCP console. Click on the multicluster-engine and select the YAML tab.
+
+Find the two image instances of the cluster-curator-controller and point to the one you built.
+
+```
+- image: >-
+        quay.io/fxiang1/cluster-curator-controller@sha256:0df33edc4662906bed71a52301597774a420e0c51bd58ec70c5a7f7600380f18
```
Comment (chenz4027) [inline docs/README.md:48]: So does this one. I'm just not sure if we would want to have the example from your personal built image.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ "configurations": [
+   {
+     "name": "Launch",
+     "type": "go",
+     "request": "launch",
+     "mode": "debug",
+     "program": "${workspaceRoot}/cmd/manager",
+     "env": {
+       "WATCH_NAMESPACE": "",
+       "IMAGE_URI": "quay.io/fxiang1/cluster-curator-controller@sha256:ce527566269f4bffad08ae8eb8533c9f829406d8bfc09299f4a84fe5492666b5"
```
[the remark this answers is not in the document]
Comment (fxiang1) [reply docs/README.md]: Yep. Good catch @chenz4027. Thanks!

**Review by chenz4027 — APPROVED** [review]
/lgtm

**Pull request merged**